pglogical_worker.c

```diff
@@ -309,6 +309,12 @@ pglogical_worker_attach(int slot, PGLogicalWorkerType type)
 	Assert(slot >= 0);
 	Assert(slot < PGLogicalCtx->total_workers);
 
+	/* Establish signal handlers. */
+	pqsignal(SIGTERM, handle_sigterm);
+
+	/* Now safe to process signals */
+	BackgroundWorkerUnblockSignals();
```

Comment:
Please add a comment mentioning the findings from RT 81571 and why these two should better be close together. So we are less likely to repeat this same mistake.

Reply:
Done. I wish Postgres would have had that comment somewhere in its code.